x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/index.tsx

```diff
@@ -221,6 +224,7 @@ const ActionsComponent: React.FC<ActionProps> = ({
             />
             <PinEventAction
               ariaLabel={i18n.PIN_EVENT_FOR_ROW({ ariaRowindex, columnValues, isEventPinned })}
+              isAlert={isAlert(ecsData)}
```

Comment:
Since `eventType` is already extracted above (line 108), I wonder if it might be better to change `isAlert(ecs: Ecs)` to `isAlert(eventType: string)`. That way you'd save one `getEventType` call per render.